habitat_sim/simulator.py

```diff
@@ -473,28 +445,23 @@ def step_physics(self, dt: float, scene_id: int = 0) -> None:
         self.step_world(dt)
 
 
-class Sensor:
-    r"""Wrapper around habitat_sim.Sensor
-
-    TODO(MS) define entire Sensor class in python, reducing complexity
-    """
-
-    def __init__(self, sim: Simulator, agent: Agent, sensor_id: str) -> None:
-        self._sim = sim
-        self._agent = agent
+class SensorReader:
```

Comment:
IMO it still makes sense to have the simulator drive the drawing process.  This class can then be mostly stateless and dumb then.

Reply:
Thanks Erik, that's fixed in the newest PR. I'm having some trouble with "SensorReader" being undefined though, and I'm not sure how to fix it. Do you have any suggestions on what I'm missing?

Reply:
I think you need to put the SensorReader name in quotes for the typing as that symbol hasn't been defined yet